src/helpers/build/mod.rs

```diff
@@ -16,37 +14,76 @@ const LIB_EXT: &str = "so";
 const LIB_EXT: &str = "dylib";
 
 fn get_output_path() -> PathBuf {
-    let manifest_dir_string = env::var("CARGO_MANIFEST_DIR").unwrap();
-    let build_type = env::var("PROFILE").unwrap();
-
-    // Workaround for case, when plugins is a subcrate of workspace
-    if !Path::new("../Cargo.toml").exists() {
-        return Path::new(&manifest_dir_string)
-            .join("target")
-            .join(build_type);
-    }
-
-    let cargo_toml_file: File = File::open("../Cargo.toml").unwrap();
-    let toml_reader = BufReader::new(cargo_toml_file);
-    for line in toml_reader.lines() {
-        let line = line.unwrap();
-        if line.contains("workspace") {
-            return Path::new(&manifest_dir_string)
-                .join("..")
-                .join("target")
-                .join(build_type);
-        }
-    }
-
-    Path::new(&manifest_dir_string)
-        .join("target")
-        .join(build_type)
+    // Get output path from env variable to get path up until debug/ or release/ folder
+    // Not proud with this code
+    Path::new(&env::var("OUT_DIR").unwrap())
+        .parent()
+        .unwrap()
+        .parent()
+        .unwrap()
+        .parent()
+        .unwrap()
+        .to_path_buf()
```

Comment:
Должно примерно так сработать

```suggestion
    Path::new(&env::var("OUT_DIR").unwrap())
        .iter()
        .nth_back(2)
        .unwrap()
        .into()
```

Reply:
Почитал доку Path, есть схема работы с этой штукой
```
Path::new(&env::var("OUT_DIR").unwrap())
        .ancestors()
        .take(4)
        .collect()
```

Reply:
Огонь, а я этот метод не заметил